src/multisig/multisig.questions.ts

```diff
@@ -0,0 +1,25 @@
+import { Question, QuestionSet } from 'nest-commander';
+
+import {
+    MULTISIG_QUESTION_MESSAGE,
+    MULTISIG_QUESTION_NAME,
+    MULTISIG_QUESTION_SET_NAME,
+    MULTISIG_QUESTION_TYPE
+} from '@/multisig/multisig.questions.descriptor';
+import { ethers } from 'ethers';
+
+export type MultisigAnswers = {
+    [MULTISIG_QUESTION_NAME]: string;
+};
+
+@QuestionSet({ name: MULTISIG_QUESTION_SET_NAME })
+export class MultisigQuestions {
+    @Question({
+        type: MULTISIG_QUESTION_TYPE,
+        name: MULTISIG_QUESTION_NAME,
+        message: MULTISIG_QUESTION_MESSAGE
+    })
```

Comment:
If the user reached this question, they selected a network and are ready to put in an address - so let's make this question `required` instead of `optional`.

If I'm not mistaken, we can achieve this by adding `validate: (value: string) => ethers.isAddress(value)`. Can you please look into it?